src/integration.test.ts

```diff
@@ -0,0 +1,25 @@
+import { Settings } from "luxon";
+
+import BugzillaAPI from ".";
+
+// Force all times to UTC.
+Settings.defaultZone = "UTC";
+
+// Tests that rely on the network are not ideal but it's hard to otherwise
+// verify that we can parse data from the real services.
+
+test("Mozilla", async () => {
+  let api = new BugzillaAPI("https://bugzilla.mozilla.org/");
+
+  await expect(
+    api.getBugs([645699, 1743832, 1749908]),
+  ).resolves.toMatchSnapshot();
+});
+
+test("Red Hat", async () => {
+  let api = new BugzillaAPI("https://bugzilla.redhat.com/");
+
+  await expect(
+    api.getBugs([1944441, 1749908, 1205830]),
```

Comment:
Those Bugs are great, but I would like to see also some bugs for `RHEL` (e.g. `1906064, 1730084`). Thank You!

Reply:
@Mossop Also I'm thinking if it would be beneficial to add some `private` bugs, for testing purposes.

Reply:
I understand, Thanks